index.js

```diff
@@ -157,6 +182,22 @@ class DetailsDialogElement extends HTMLElement {
     })
   }
 
+  get src(): ?string {
+    return this.getAttribute('src')
+  }
+
+  set src(value: string) {
+    value ? this.setAttribute('src', value) : this.removeAttribute('src')
```

Comment:
I think we should only `removeAttribute` on boolean values. Setting the property to an empty string is fine.